### ISSUE TITLE
read.neurons.catmaid could add option for annotation or neuron name query

### ISSUE THREAD

**Issue by jefferis** [jefferis/rcatmaid#10]
note that by chance `nat::read.neurons` has a `neuronnames` argument

**Comment by jefferis** [comment]
perhaps anything that expects one or more skids (as first input), could process the input with a function like 
```r
catmaid_skeleton_ids<-function(x, several.ok=TRUE) {
  if(is.factor(x)) {
   x=as.character(x)
  }
  if(is.character(x)) {
    # check if these look like skids, convert to int
   skids = as.integer(skids)
    # ... otherwise find skids with a query (by name or annotation?)
    "name:",     "annotation:" 
    skids = df$skid
  } else 
  # check that skids several.ok is matched
}
```

**Comment by jefferis** [comment]
could also accept input like c(name='a', annot='b', annot='c')

**Comment by jefferis** [comment]
closed by 8ba6506344f2f944f9861eab2034b8911c1e659f and merged in 08c1a6ef78384b0dec33a623cfff01e43f4a2c41.
